include/gsl/gsl_assert

```diff
@@ -68,20 +68,54 @@
 
 namespace gsl
 {
+
 struct fail_fast : public std::runtime_error
 {
     explicit fail_fast(char const* const message) : std::runtime_error(message) {}
 };
-}
+namespace details
+{
+template <typename Exception>
+void throw_exception(Exception&& exception);
```

Comment:
I think attribute syntax is not supported by vs2013 - I ran into a similar problem with a different PR. Now that VS2017 is out I hope support for vs2013 will be dropped soon.